src/zenpos.jl

```diff
@@ -0,0 +1,78 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _zenpos{T<:AbstractFloat}(jd::T, latitude::T, longitude::T)
+
+    lst = ct2lst(longitude, jd)
+    ra = lst*pi/12
+    dec = ra*0.0 + deg2rad(latitude)
```

Comment:
Yeah, it was unnecessary. But I saw the same being done in geo2mag.jl and mag2geo.jl (Line 9 in both files), so I let it stay.